lib/charms/prometheus_k8s/v0/prometheus_remote_write.py

```diff
@@ -1136,7 +1136,7 @@ def _apply_label_matcher(self, expression, topology):
         try:
```

Comment:
LIBPATCH ok?

Reply:
Seems like I forgot to save that file. It is now. 🙏🏼